alpha/lib/model/kuserPeer.php

```diff
@@ -502,6 +502,20 @@ public static function addUser(kuser $user, $password = null, $checkPasswordStru
 		// if password is set, user should be able to login to the system - add a user_login_data record
 		if ($password || $user->getIsAdmin()) {
 			// throws an action on error
+			$allowedEmailDomainsForAdmins = explode(',', $partner->getAllowedEmailDomainsForAdmins());
+
+			if (!kString::isEmailString($user->getEmail()))
```

Comment:
i preffer that we will check it inside the following if (after line 513)
for backword cometability